testcases/OpTestEEH.py

```diff
@@ -379,3 +385,18 @@ def check_eeh_pe_recovery(self, pe):
             if pe in device:
                 return True
         return False
+
+
+    ##
+    # @brief  This function is used to get count of interfaces in PCI devices
+    #         available at that instant of time.
+    #
+    # @returns count @type int: interface count
+    #
+    def get_count_of_interdaces_in_pci_devices(self, pci_addr):
+        cmd = "ls -l /sys/class/*/ -1"
```

Comment:
This command is trying to get all the interfaces in the system. Seeing huge console output. Can you optimize the command output by using some regular expressions or some thing else.